src/flb_log.c

```diff
@@ -780,3 +803,61 @@ int flb_log_destroy(struct flb_log *log, struct flb_config *config)
 
     return 0;
 }
+
+struct flb_log_metrics *flb_log_metrics_create()
+{
+    struct flb_log_metrics *lm;
+    int i;
+    const char *severity_str;
+    uint64_t ts;
+
+    lm = flb_calloc(1, sizeof(struct flb_log_metrics));
+    if (!lm) {
+        flb_errno();
+        goto error;
+    }
+
+    lm->cmt = cmt_create();
+    if (!lm->cmt) {
+        goto error;
+    }
+
+    lm->logs_total_counter = cmt_counter_create(lm->cmt,
+                                                "fluentbit", "runtime", "logs_total",
```

Comment:
We seem to have very few internal metrics so I'm not sure if there's a naming convention aside from the `fluentbit_` prefix. Maybe this could just be `fluentbit_logger_logs_total`?

Reply:
Went with `fluentbit_logger_logs_total` for now.